packages/h/hffix/xmake.lua

```diff
@@ -0,0 +1,17 @@
+package("hffix")
+
+    set_homepage("https://github.com/jamesdbrock/hffix")
```

Comment:
we need set headeronly kind

see https://github.com/xmake-io/xmake-repo/blob/c1f4645941199156d5a1836267d4597bd2364acc/packages/d/dr_mp3/xmake.lua#L3

Reply:
Thanks, fixed.